content/en/getting-started/installation.md

```diff
@@ -1,155 +1,206 @@
-
 {{< command >}} 
 $ python3 -m pip install localstack==<version>
 {{< / command >}}
-
+{{% markdown %}}
 Here `<version>` depicts the particular LocalStack version that you would like to download and use.
+{{% /markdown %}}
 {{< /alert >}}
 
 {{< alert title="Important" color="danger" >}}
 Do not use `sudo` or the `root` user - LocalStack should be installed and started entirely under a local non-root user. 
-If you have problems with permissions in MacOS X Sierra, install with `python3 -m pip install --user localstack`.
 {{< /alert >}}
+{{< /tab >}}
+{{< /tabpane >}}
 
-Afterwards you should be able to use the LocalStack CLI in your terminal:
-
-{{< command >}}
-$ localstack --help
-Usage: localstack [OPTIONS] COMMAND [ARGS]...
-
-  The LocalStack Command Line Interface (CLI)
-
-Options:
-...
-{{< / command >}}
-
-#### Updates
-
-The LocalStack CLI also allows you to easily update the different components of LocalStack. To check the various options available for updating, run:
-
+To verify that the LocalStack CLI was installed correctly, you can check the version in your terminal:
 {{< command >}}
-$ localstack update --help
-
-Usage: localstack update [OPTIONS] COMMAND [ARGS]...
-
-  Update LocalStack components
-
-Options:
-  --help  Show this message and exit.
-
-Commands:
-  all             Update all LocalStack components
-  docker-images   Update container images LocalStack depends on
-  localstack-cli  Update LocalStack CLI tools
+$ localstack --version
+{{< localstack-latest-version >}}
 {{< / command >}}
 
-You can decide to update the CLI itself, the LocalStack Docker images, or all at once:
+You are all set!
+To use all of LocalStack's features we recommend to [get a LocalStack account and set up your api key]({{< ref "api-key" >}}).
+Afterwards, check out our [Quickstart guide]({{< ref "quickstart" >}}) to start your local cloud!
 
-{{< command >}}
-$ # Update all components
-$ localstack update all
-$ # Only update the LocalStack docker images
-$ localstack update docker-images
-$ # Only update the LocalStack CLI
-$ localstack update localstack-cli
-{{< / command >}}
-
-#### Starting LocalStack with the LocalStack CLI
+## Alternatives
+Besides using the CLI, there are other ways of starting and managing your LocalStack instance:
 
-By default, LocalStack is started inside a Docker container by running:
+- [LocalStack Cockpit]({{< ref "#localstack-cockpit" >}})\
+  Get a desktop experience and work with your local LocalStack instance via the UI.
 
-{{< command >}}
-$ localstack start
-{{< / command >}}
+- [LocalStack Docker Extension]({{< ref "#localstack-docker-extension" >}})\
+  Use the LocalStack extension for Docker Desktop to work with your LocalStack instance.
 
-{{< alert title="Notes" >}}
-- This command loads all services provided by LocalStack, they will however be started on the first request reaching this service.
+- [Docker]({{< ref "#docker" >}})\
```

Comment:
👉 This option comes with so many limitations (e.g., no caching, services that require the Docker socket won't work) that we might want to de-emphasize it by moving further down the list. Hence, Docker-compose should clearly appear first in this list.

Reply:
That is true, it makes sense to push docker-compose in this case. I moved Docker down below docker-compose and rephrased the two section intros a bit.